jms/org.wso2.carbon.transport.jms/src/main/java/org/wso2/carbon/transport/jms/factory/JMSConnectionFactory.java

```diff
@@ -493,7 +490,6 @@ public MessageConsumer createMessageConsumer(Session session, Destination destin
      * @param session     JMS Session to create the producer
      * @param destination JMS destination which the producer should publish to
      * @return MessageProducer, who publish messages to particular destination with the given session
-     * @throws JMSConnectorException JMS Connector Exception
```

Comment:
Method signature has "throws JMSConnectorException", then why do we remove it form the doc?

Reply:
This is fixed. The exception is added.